### ISSUE TITLE
Feature ipv6 not supported with method pf

### ISSUE THREAD

**Issue by matiwinnetou** [sshuttle/sshuttle#84]
shuttle version: 0.77.2
```
W sshuttle -vv -H --dns -r pi@matiwinnetou.no-ip.org 192.168.1.0/24
Starting sshuttle proxy.
firewall manager: Starting firewall with Python version 2.7.10
firewall manager: ready method name pf.
fatal: Feature ipv6 not supported with method pf.
```

**Comment by brianmay** [comment]
Believe this is the same as #74 which has been fixed, however not released yet.

**Comment by brianmay** [comment]
Released version 0.78.0; closing.
